### PR TITLE
fix(security): remove command test execution - CVE 2019-16405

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

* Remove command test execution
* Remove dead code
* Apply sanitize on arguments
* Use camel case
* Use PDO:bindParam

**Fixes** CVE 2019-16405, thank to TheCyberGeek and Khaled

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [x] 18.10.x
- [x] 19.04.x
- [x] 19.10.x (master)

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/include/configuration/configObject/command/formCommand.ihtml [hunk 1]
```diff
@@ -53,8 +53,8 @@
 			</td>
 		</tr>
         <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="enable_shell"> {$form.enable_shell.label}</td><td class="FormRowValue">{$form.enable_shell.html}</td></tr>
-		<tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="arg_example"> {$form.command_example.label}</td><td class="FormRowValue">{if $o != "w"}{$form.command_example.html} &nbsp;{$form.command_hostaddress.label}&nbsp;{$form.command_hostaddress.html}&nbsp;<img alt='{$cmd_play}' title='{$cmd_play}' src='./img/icones/16x16/media_play.gif' onClick="window.open('main.php?p=60801&command_hostaddress='+ document.Form.command_hostaddress.value +'&command_example='+ document.Form.command_example.value +'&command_line='+ document.Form.command_line.value + '&o=p&min=1','','toolbar=no,location=no,directories=no,status=no,scrollbars=yes,resizable=yes,copyhistory=no, width=700, height=400');">{/if}</td></tr>
-		<tr class="list_one">
+        <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="arg_example"> {$form.command_example.label}</td><td class="FormRowValue">{if $o != "w"}{$form.command_example.html}{/if}</td></tr>
```
Comment (kduret) [inline www/include/configuration/configObject/command/formCommand.ihtml]: bad indentation

File: www/include/configuration/configObject/command/minHelpCommand.php [hunk 2]
```diff
@@ -38,56 +38,69 @@
...
 } elseif (isset($_POST["command_id"])) {
-    $command_id = $_POST["command_id"];
+    $commandId = $_POST["command_id"];
 } else {
-    $command_id = null;
+    $commandId = null;
 }
 
+$commandId = filter_var(
+    $commandId ?? null,
```
Comment (kduret) [inline www/include/configuration/configObject/command/minHelpCommand.php]: I think you don't need to put "?? null"

File: www/include/configuration/configObject/command/minHelpCommand.php [hunk 3]
```diff
@@ -38,56 +38,69 @@
...
-    $command_name = $_POST["command_name"];
+    $commandName = $_POST["command_name"];
 } else {
-    $command_name = null;
+    $commandName = null;
 }
 
-if ($command_id != null) {
+$commandName = filter_var(
+    $commandName ?? null,
```
Comment (kduret) [inline www/include/configuration/configObject/command/minHelpCommand.php]: I think you don't need to put "?? null"

File: www/include/configuration/configObject/command/minHelpCommand.php (hunk 2 above)
Comment (kduret) [reply www/include/configuration/configObject/command/minHelpCommand.php]: furthermore, it's not compatible with PHP 5.4 (2.8.x)

**Comment by kduret** [comment]
Furthermore, this is not cherry-pickable on 2.8.x cause usage of bindParam and ??

**Comment by lpinsivy** [comment]
I will made another one for 2.8.x

**Pull request merged**